README.md

```diff
@@ -50,6 +56,7 @@ In order to support the six bed functions, at least six relays are required. How
 
 ### Hand Control
 
+---> I think a photo showing the cable connected from relay to hand control would be useful, especially for hardware ignorant people.
```

Comment:
Yes, we need to get in touch with Isaac and see if he can get us a picture that will serve this purpose until we can get a video that goes into more detail.